xc7/utils/prjxray_edge_library.py

```diff
@@ -979,7 +1006,7 @@ def make_connection(
 
     pip_obj = find_pip(tile_type, pip.name)
 
-    assert not pip_obj.is_pseudo
+    assert not pip_obj.is_pseudo or "CLK_HROW_CK" in pip.name
```

Comment:
Let's add a comment on what is going on here

Reply:
Done